app/src/main/kotlin/com/mapbox/vision/teaser/replayer/SessionsAdapter.kt

```diff
@@ -1,42 +1,180 @@
 package com.mapbox.vision.teaser.replayer
 
+import android.annotation.SuppressLint
+import android.content.Context
 import android.view.LayoutInflater
 import android.view.View
+import android.view.View.INVISIBLE
+import android.view.View.VISIBLE
 import android.view.ViewGroup
+import android.widget.ImageView
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 import com.mapbox.vision.teaser.R
+import com.mapbox.vision.teaser.replayer.SessionsAdapter.AdapterItem.CameraItem
+import com.mapbox.vision.teaser.replayer.SessionsAdapter.AdapterItem.SessionItem
 import java.io.File
+import java.text.SimpleDateFormat
+import java.util.*
 
-class SessionsAdapter(basePath: String, val clickListener: (String) -> Unit) :
-    RecyclerView.Adapter<SessionsAdapter.SessionViewHolder>() {
+class SessionsAdapter(private val context: Context,
```

Comment:
nit : to align indents in such cases we use  
```
class SessionsAdapter(
   ...
) : Parent
```
styling

Reply:
fixed